egs/librispeech/asr/nnlm/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Xiaomi Corporation (Author: Liyong Guo)
+# Apache 2.0
+
+# References:
+# https://github.com/kaldi-asr/kaldi/blob/master/scripts/rnnlm/train_rnnlm.sh
+# https://github.com/kaldi-asr/kaldi/blob/master/scripts/rnnlm/prepare_rnnlm_dir.sh
+# https://github.com/pytorch/examples/tree/master/word_language_model
+# https://huggingface.co/docs/tokenizers/python/latest/quicktour.html
+
+# Example of how to use HuggingFace tokenizer and train {RNN, Transformer} based LMs
+
+set -e
+stage=$1
+
+lm_train=data/lm_train/
+full_text=$lm_train/librispeech_train_960_text
+tokenizer=$lm_train/tokenizer-librispeech_train_960.json
+if [ $stage -eq 1 ]; then
```

Comment:
Should it be `$stage -le  1`?
And also for the following `if` statements.

Reply:
yes. "-le" is better. Now "-eq" is used beacuse it's easy for me to debug stage by stage.